cmd/runhcs/vm.go

```diff
@@ -92,7 +92,7 @@ var vmshimCommand = cli.Command{
 			exitCh <- vm.Wait()
 		}()
 
-		defer vm.Terminate()
```

Comment:
Note: This would not wait if `vm.Terminate()` returned pending.
Note2: We don't use `runhcs.exe` anymore so this is just a minor cleanup. Since it was the last user I thought just remove `uvm.Terminate` to avoid any other callers since `.Close()` handles this case already.